database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
+  const businesses = [];
+  for (let i = 0; i < 100; i += 1) {
+    const entry = {
+      name: faker.company.companyName(),
+      claimed: Math.floor(Math.random() * Math.floor(2)),
+      // overallRating needs future refactor to generate dynamic rating
+      overallRating: Math.floor(Math.random() * Math.floor(5)) + 1,
+      // totalReviews needs future refactor to generate dynamic count
+      totalReviews: Math.floor(Math.random() * Math.floor(60)) + 10,
+      averageCost: Math.floor(Math.random() * Math.floor(4)) + 1,
+      businessType: `${faker.company.catchPhraseAdjective()}, ${faker.name.jobArea()}`,
+      address1: `${faker.address.streetAddress()}`,
+      address2: `${faker.address.city()}, ${faker.address.stateAbbr()} ${faker.address.zipCode()}`,
+      address3: `b/t ${faker.address.streetName()} ${faker.address.streetSuffix()} & ${faker.address.streetName()} ${faker.address.streetSuffix()}`,
```

Comment:
Maybe there are better options for these variable name such as 'City, ZipCode" etc which are more descriptive. This will make it easier for you to remember which is which later on.

Reply:
I was looking at those. I refactored my code to use descriptive variable names. I had gone with address1, etc. while I was rushing through something else.